module/VuFind/src/VuFind/Search/Factory/EdsBackendFactory.php

```diff
@@ -103,6 +110,8 @@ public function __invoke(ContainerInterface $sm, $name, array $options = null)
         $this->setup($sm);
         $this->edsConfig = $this->getService(\VuFind\Config\PluginManager::class)
             ->get($this->getServiceName());
+        $this->vuFindConfig = $this->getService(\VuFind\Config\PluginManager::class)
```

Comment:
This is no longer used, so I think you can remove the assignment and property from the factory.